docs/development.md

```diff
@@ -1,5 +1,13 @@
 ## Development
 
+### Environment Variables
+
+Default environment variables are declared in `env.default`. If you wish to override any of the values, you can create a local `.env` file in the root of the project. This file should not be committed.
```

Comment:
We have `.env` in our `.gitignore` so we shouldn't have to say that it shouldn't be committed (even if people try, git won't do it)

```suggestion
Default environment variables are declared in `env.default`. If you wish to override any of the values, you can create a local `.env` file in the root of the project.
```